examples/gallery/lines/contours.py

```diff
@@ -0,0 +1,41 @@
+"""
+Contours
+--------
+The :meth:`pygmt.Figure.contour` method can plot contour lines from a table of points by direct triangulation.
+The data to the triangulation can by provided in one of three options:
+    1: ``x``, ``y``, ``z`` 1d :class:`numpy.ndarray` data columns\n
+    2: ``data`` 2d :class:`numpy.ndarray` data matrix with 3 columns corresponding to ``x``, ``y``, ``z``\n
+    3: ``data`` path string to a file containing the ``x``, ``y``, ``z`` in a tabular format\n
```

Comment:
```suggestion

#. ``x``, ``y``, ``z`` 1d :class:`numpy.ndarray` data columns.
#. ``data`` 2d :class:`numpy.ndarray` data matrix with 3 columns corresponding
   to ``x``, ``y``, ``z``.
#. ``data`` path string to a file containing the ``x``, ``y``, ``z`` in a
   tabular format.

```
This suggestion applies ReStructuredText formatting to the list.